data/presets/man_made/charge_point.json

```diff
@@ -0,0 +1,33 @@
+{
+    "icon": "fas-charging-station",
+    "fields": [
+        "brand",
+        "operator",
+        "capacity",
+        "access_simple",
+        "fee",
+        "payment_multi_fee",
+        "charge_fee",
+        "ref"
+    ],
+    "moreFields": [
+        "covered",
+        "level",
+        "manufacturer",
+        "maxstay",
+        "network"
+    ],
+    "geometry": [
+        "point"
+    ],
+    "tags": {
+        "man_made": "charge_point"
+    },
+    "terms": [
+        "ev",
+        "electric vehicle",
+        "charging station",
+        "charge point"
+    ],
+    "name": "EV Charging Stall"
```

Comment:
Isn't _charge point_ the more common term for these devices in English?

Reply:
I think that charge point is more of a British term[^1][^2][^3], at least on the west coast these are usually referred to as charging stalls[^4][^5][^6]

[^1]
[^2]
[^2]

Reply:
~~I think that charge point is more of a British term, these are usually called charge stalls on the west coast of the US~~

Never mind, it's probably best to use 'point' since 'stall' could refer to both the charger and parking space. I'll change the label to _charging point_ instead of _charge point_ to disambiguate this preset from the brand [ChargePoint](https://www.chargepoint.com/)